<?xml version="1.0"?>
<launch>
    <!-- CONTROLLER -->
	<param
        name="quadrotor"
        value="$(find awesomo)/configs/quadrotor/config.yaml" />
	<param
        name="position_controller"
        value="$(find awesomo)/configs/position_controller/config.yaml" />
	<param
        name="carrot_controller"
        value="$(find awesomo)/configs/carrot_controller/config.yaml" />
	<node
	    pkg="awesomo"
	    type="awesomo"
	    name="awesomo"
        output="screen" />
</launch>
